app/models/reminder.rb

```diff
@@ -50,6 +58,11 @@ def self.conditionally_create_reminder_for_volunteer(volunteer, assignment)
     end
   end
 
+  def self.conditionally_create_trial_end_reminder_for_volunteer(volunteer, a)
+    Reminder.create!(volunteer: volunteer, assignment: a, kind: 0) if a.started_ca_six_weeks_ago? &&
```

Comment:
would prefer to use `assignment` instead of only using `a`. didn't abbreviated for the methods above, and its clearer.

Reply:
Me too, I did that because RuboCop complains about the line length.